src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -0,0 +1,188 @@
+    dao.create(scheduledInterview2);
+    List<ScheduledInterview> result = dao.getForPerson("user@company.org");
+    dao.delete(result.get(0).id());
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = dao.entityToScheduledInterview(entity);
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+    Assert.assertEquals(copyScheduledInterview2, storedScheduledInterview);
+  }
+
+  // Tests updating a user's scheduledInterview.
+  @Test
+  public void updatesScheduledInterview() {
+    dao.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview previousStoredScheduledInterview = dao.entityToScheduledInterview(entity);
+    ScheduledInterview updatedStoredScheduledInterview =
+        ScheduledInterview.create(
+            previousStoredScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user3@mail.com");
+    dao.update(updatedStoredScheduledInterview);
+    Entity updatedEntity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview updatedScheduledInterview = dao.entityToScheduledInterview(updatedEntity);
+    Assert.assertEquals(updatedStoredScheduledInterview, updatedScheduledInterview);
+  }
+
+  // Tests retrieving a scheduledInterview from Datastore.
+  @Test
+  public void getsScheduledInterview() {
+    dao.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = dao.entityToScheduledInterview(entity);
+    Optional<ScheduledInterview> actualScheduledInterviewOptional =
+        dao.get(storedScheduledInterview.id());
+    ScheduledInterview expectedScheduledInterview =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    Optional<ScheduledInterview> expectedScheduledInterviewOptional =
+        Optional.of(expectedScheduledInterview);
+    Assert.assertEquals(expectedScheduledInterviewOptional, actualScheduledInterviewOptional);
+  }
+
+  // Tests retrieving a scheduledInterview that doesn't exist from Datastore.
+  @Test
+  public void failsGetScheduledInterview() {
+    Optional<ScheduledInterview> actual = dao.get(2);
+    Optional<ScheduledInterview> expected = Optional.empty();
```

Comment:
This line is unnecessary. Just do `Assert.assertEquals(Optional.empty(), actual);`